simpegEM/FDEM/FDEM.py

```diff
@@ -221,8 +322,9 @@ def getA(self, freq):
         mui = self.MfMui
         sigI = self.MeSigmaI
         C = self.mesh.edgeCurl
+        iomega = 1j * omega(freq) * sp.eye(self.mesh.nF)
```

Comment:
Good call on iomega only multiplying the diagonal. I don't think i would have caught that right away...